src/ol-maplibre-layer.ts

```diff
@@ -6,24 +6,93 @@ import {Map as MaplibreMap} from 'maplibre-gl';
 import type {FrameState} from 'ol/Map.js';
```

Comment:
Please remove not used import (this one, and `toDegrees` and `toLonLat` above)

Reply:
Done